### PR TITLE
feat(MA-10): open new account.

### PR DESCRIPTION
**Pull request by antoinemeyer**
https://trizic.atlassian.net/browse/MA-10

### CODE REVIEW

**Comment by antoinemeyer** [comment]
not based on #6 anymore as it seems there are some discussions on onboarding flow status.
@johntowell @olmstead @eweise please review

File: api/v3/examples/accountsPOST.req.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "profileId": "69104639-084a-11d6-9b9b-0ef9454abd2e",
```
Comment (johntowell) [inline api/v3/examples/accountsPOST.req.json]: Do we use all of this data? If we are only storing it to populate docusign we should think about just using the docusignFormData passthrough instead?

Comment (antoinemeyer) [inline api/v3/examples/accountsPOST.req.json]: we don't use that data (should not even store it).
if we use the docusign form data passthrough, how does the client know what fields to send? doesn't it defeat the purpose of this raml spec?

Comment (olmstead) [inline api/v3/examples/accountsPOST.req.json]: I agree that if we know what fields we are expecting, then we should validate.  However, there are interdependencies between the field, for instance joint account info is only required joint account is selected, so the validation is dependent on business logic.  The only way I see to resolve that is if we have separate endpoints base on account type, but I don;t think we want to go down that route.

Comment (antoinemeyer) [inline api/v3/examples/accountsPOST.req.json]: ok, I'll see what we really need to store if any and remove most of those fields to just have them forwarded to docusign. I'll update the PR once done.

Comment (olmstead) [inline api/v3/examples/accountsPOST.req.json]: @antoinemeyer @abuecker I think we still want to validate on our side before sending on.  Can we  chat after standup today to discuss the idea of separate routes for each account type?

Comment (johntowell) [inline api/v3/examples/accountsPOST.req.json]: It sounds like JH will have their own docusign account, and be able to create their own templates, so I'm not sure we can validate the input. 

Potentially we could do some sort of smart validation where we query the docusign templates and validate that we have data for all of the given docusign form datas. Although that assumes we never want the user to fill out any docusign forms.

Comment (olmstead) [inline api/v3/examples/accountsPOST.req.json]: Ah right, once they start creating their own consumer client they can send anything.  Sounds like we need to stick with John's generic solution for now. Let's discuss with Andy after standup.

Comment (johntowell) [inline api/v3/examples/accountsPOST.req.json]: Also these templates (and form data) change every year, so I think it is nice to have a flexible api to accommodate that.

I do like the ability for a given combination of account type, funding type to determine what form 
data needs to be passed in, and then validating that. I can research if that is possible with Docusign API.

Comment (olmstead) [inline api/v3/examples/accountsPOST.req.json]: I'm finding it difficult to review code that is not tied to andy requirements.  Unless I'm looking in the wrong place, I don't see even a description for MA-10. I think we need to some requirements to work off before coding or opening a PR.

**Comment by antoinemeyer** [comment]
I've cleaned up the fields that can just be sent directly to docusign 
@johntowell @olmstead @eweise please review

**Comment by antoinemeyer** [comment]
@johntowell @olmstead @eweise please review

**Comment by johntowell** [comment]
👍

**Pull request merged**